src/coreclr/tools/Common/Compiler/DependencyAnalysis/ObjectDataBuilder.cs

```diff
@@ -85,42 +86,37 @@ public void EmitByte(byte emit)
 
         public void EmitShort(short emit)
         {
-            EmitByte((byte)(emit & 0xFF));
-            EmitByte((byte)((emit >> 8) & 0xFF));
+            Span<byte> buffer = stackalloc byte[sizeof(short)];
```

Comment:
Do you have any perf vnumbers that show this is more efficient?

Reply:
[Got my benchmarks done ](https://github.com/dotnet/runtime/pull/102394#issuecomment-2118414551) 👍 stackalloc did okay-ish, switched to better alternative.